docs/cli/geo/maps.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Maps
+description: Use Amplify CLI to create and manage maps to visualize GeoSpatial data in your app.
+---
+
+Amplify CLI's `geo` category enables you to create and manage map resources used to visualize GeoSpatial data in your application.
+You need to install Amplify CLI with the `@geo` tag in order to get the Geo functionality. You can use the following command to install this version globally.
+```console
+npm i -g @aws-amplify/cli@geo
+```
+
+## Setup a new Map
+
+You can add a new map by running the following command from your project's root folder:
+
+```bash
+amplify add geo
+```
+```console
+? Select which capability you want to add:
+> Map (visualize the GeoSpatial data)
+  Location search (search by places, addresses, coordinates)
+```
+If you haven't set up the `auth` category already, the Amplify CLI will guide you through a workflow to enable the auth category.
+
+```console
+? You need to add auth (Amazon Cognito) to your project in order to add geo resources. Do you want to add auth now?
+> Yes
+
+Using service: Cognito, provided by: awscloudformation
+```
+
+Next, you can set a friendly name for the Map being provisioned. 
+```console
+? Provide a name for the Map:
+> StreetsMap
+```
+
+### Map Access permissions
+
+Next, configure the access permissions for your Map resource. You can scope permissions based on an individual user's authentication status.
+
+```console
+? Who can access this Map?
+❯ Authorized users only
+  Authorized and Guest users
+```
+
+Select `Authorized users only` if only authenticated users can have read-only permissions for the map.
+
+Select `Authorized and Guest users` if both authenticated and un-authenticated users can have read-only permissions for the map.
+
+These read-only permissions allow rendering of the Map in user's application:
+```
+geo:GetMapStyleDescriptor,
+geo:GetMapGlyphs,
+geo:GetMapSprites,
+geo:GetMapTile
+```
+For more information, refer [link to location service page](https://docs.aws.amazon.com/location/latest/developerguide/security_iam_id-based-policy-examples.html#security_iam_id-based-policy-examples-get-map-tiles).
+
+### Map Pricing Plan
+Amplify provides a set of questions to help you decide the pricing plan relevant to your application's use-case. 
+These pricing plan related set of questions are presented to you when you add the first Geo resource to your application. 
+Once you chose the pricing plan, it will be automatically used for the subsequent Geo resources added to your application.
```

Comment:
First tell me why a pricing plan is even important to me. Then say "Amplify helps you determine the best pricing plan for you by guiding you through a set of questions... etc."

Reply:
added "Amazon Location Service offers three pricing plans namely, `RequestBasedUsage`, `MobileAssetTracking` and `MobileAssetManagement` that require no up-front commitment, and no minimum fee.
You select one of the following pricing plans for the Amazon Location Service resources you create for your application.
"